docs/source/installation.rst

```diff
@@ -36,6 +36,8 @@ in order for Manim to work properly, some additional system
 dependencies need to be installed first. The following pages have
 operating system specific instructions for you to follow.
 
+Manim is **only** compatible with Python versions `3.7 - 3.9`, but not `3.10` for now.
```

Comment:
Very minor nitpick: The dash in between `3.7` and `3.9` should be an en dash, not a regular hyphen. There should also not be a space before and after the en dash. The other instances should also be fixed.

Reply:
Solved, I changed all hyphens to en dashes, and removed the spaces.